src/patternfly/components/Chips/chip-group.scss

```diff
@@ -0,0 +1,48 @@
+@import "../../patternfly-utilities";
+
+.pf-c-chip-group {
+  --pf-c-chip-group--PaddingTop: calc(#{var(--pf-global--spacer--xs)} / 2);
+  --pf-c-chip-group--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--PaddingBottom: calc(#{var(--pf-global--spacer--xs)} / 2);
+  --pf-c-chip-group--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--BorderRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-chip-group--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--MarginBottom: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--BackgroundColor: transparent;
+  --pf-c-chip-group--m-toolbar--BackgroundColor: var(--pf-global--BackgroundColor--300);
+  --pf-c-chip-group--m-select-BackgroundColor: transparent;
+  --pf-c-chip-group--m-select-PaddingBottom: var(--pf-global--spacer--xs);
+  --pf-c-chip-group--m-select-Width: #{pf-size-prem(375px)}; // for demo
+
+  // Label in group
+  --pf-c-chip-group__label--PaddingLeft: var(--pf-global--spacer--xs);
+  --pf-c-chip-group__label--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-chip-group__label--FontSize: var(--pf-global--FontSize--sm);
+
+  display: inline-flex;
+  flex-wrap: wrap;
+  align-items: center;
+  padding-top: var(--pf-c-chip-group--PaddingTop);
+  padding-right: var(--pf-c-chip-group--PaddingRight);
+  padding-bottom: var(--pf-c-chip-group--PaddingBottom);
+  padding-left: var(--pf-c-chip-group--PaddingLeft);
+  margin-right: var(--pf-c-chip-group--MarginRight);
+  margin-bottom: var(--pf-c-chip-group--MarginBottom);
+  overflow: hidden;
```

Comment:
is this needed? I've noticed that with two chip groups in Firefox when they wrap this causes extra vertical space between the two groups.